doc/nrf/links.txt

```diff
@@ -219,6 +219,10 @@
 
 .. _`BH1749NUC-E`: https://www.rohm.com/datasheet/BH1749NUC/bh1749nuc-e
 
+.. _`JSON`: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/working-with-json.html
```

Comment:
```suggestion
.. _`JSON`: https://www.json.org/json-en.html
```

I think we can assume that developers know what JSON is, but if you must provide a link, consider linking to the spec, instead of the AWS Docs section of the JavaScript SDK because that guide contains information that is not relevant to IoT ("JSON as Service Object Parameters", and "Returning Data as JSON").